### DISCUSSION TITLE
Add a new workflow when a Pull Request is created in order to add the labels and the assignee of the related issue.

### DISCUSSION THREAD

**Discussion by xserrat** [pheature-flags/pheature-flags#152] [Ideas]
Checking our current process, maybe we could add a new workflow when a Pull Request is created in order to add the labels and the assignee of the related issue. 

However, the main problem here is that we'll force the contributor to add in the PR title the code of the issue in order to obtain the labels and assignees of that issue.

What do you think?

**Comment by kpicaza** [comment]
It should be great to add it to the contributing guidelines, maybe in two parts, the technical part, rebase, tests, docs, and so on, and on another hand the labeling policy. I don't know if is possible to automate this,

**Reply by xserrat** [reply]
I'll take a look if we can automate it with Github Actions!
